### ISSUE TITLE
Dmd builds fail on Linux

### ISSUE THREAD

**Issue by abaga129** [AuburnSounds/Dplug#209]
I'm using dmd v2.077.0

There error it gives is `../../core/dplug/core/sync.d(438,20): Error: variable t used before set`

```D
        else version( Posix )
        {
                timespec t = void;

                assumeNothrowNoGC(
                (timespec t, Duration period)
                {
                    mktspec( t, period );
                })(t, period); //Line 438
```

This is probably a simple fix but I don't quite understand the code.  I will investigate it though.  
It isn't of utmost importance since ldc builds dplug fine.

**Comment by p0nce** [comment]
Should be fixed in ff4a72ed4fe4fdf7e5d2e96dc14e2e7d669107f0, haven't repro on Linux though.
That's pretty cool, DMD found a real bug that thankfully didn't affected anyone.

**Comment by abaga129** [comment]
I finally had some time to test it, and dmd does build dplug on linux now. Thanks!
